packages/appstore/src/components/CFDs/cfd-real-accounts.tsx

```diff
@@ -212,6 +212,7 @@ const CFDRealAccounts = ({
                                                         amount={existing_account.display_balance}
                                                         onClickTopUp={() => onClickFundReal(existing_account)}
                                                         onClickTrade={() => {
+                                                            setAppstorePlatform(account.platform);
```

Comment:
Why do we need this?

Reply:
Initially the platform was undefined. The click method sets the platform so that it knows which platform it is.